githooks/post-merge

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# === RUST-BITCOIN GITHOOK ===
+
+# Scan the githooks after merge. If the user is using any of the rust-bitcoin githooks and they have changed, let them know
+# to rerun the githooks setup.
+
+# Get the local githooks directory, regardless of configuration.
+GIT_DIR=$(git rev-parse --git-common-dir)
+HOOKS_DIR=$(git config --get core.hooksPath || echo "$GIT_DIR/hooks")
+
+# Scan each of the hooks looking for the tag. If the tag is found, then the user is using a
+# rust-bitcoin githook. We should compare them to the repo's hooks to see if they have
+# changed in the latest git pull.
+for hook in "$HOOKS_DIR"/*
+do
+	if grep -q '=== RUST-BITCOIN GITHOOK ===' "$hook"; then
+		BN=$(basename "$hook")
+		if ! cmp --quiet "$hook" "githooks/$BN"; then
+			echo "========================================"
+			echo "Project githooks have changed."
+			echo "Please re-run \`just githooks-install\`."
```

Comment:
It should rather say "Please inspect the changes and re-run `just githooks install` if they are legitimate."